frontend/src/components/contexts/contexts.ts

```diff
@@ -1,14 +1,22 @@
 import * as React from "react";
-import { MockProfileBackendService } from 
-  "../../services/mock_profile_backend/mock_profile_backend";
-import { ProfileHandlerService } from 
+import { MockProfileBackendService } from
+  "../../services/mock_backend/mock_profile_backend";
+import { MockYourClubsBackendService } from
+  "../../services/mock_backend/mock_your_clubs_backend";
+import { MAX_NUM_CLUBS } from
+  "../../utils/temporary_testing_consts";
+import { NUM_PROFILES } from
+  "../../utils/temporary_testing_consts";
```

Comment:
```suggestion
import { MAX_NUM_CLUBS, NUM_PROFILES } from "../../utils/temporary_testing_consts";
```

Same in some other files. We should have the linter complain about this too. In general you only want to import each file once.